xmr-btc/src/config.rs

```diff
@@ -24,6 +25,7 @@ impl Config {
             // blockchain is slow
             monero_max_finality_time: (*mainnet::MONERO_AVG_BLOCK_TIME).mul_f64(1.5)
                 * mainnet::MONERO_FINALITY_CONFIRMATIONS,
+            monero_min_confirmations: mainnet::MONERO_FINALITY_CONFIRMATIONS,
```

Comment:
```suggestion
            monero_finality_confirmations: mainnet::MONERO_FINALITY_CONFIRMATIONS,
```
Please use the same terminology across the codebase.